### PR TITLE
upgrade to logstash_v5

### PR DESCRIPTION
**Pull request by zahodi**

### CODE REVIEW

File: tasks/setup-Debian.yml [hunk 1]
```diff
@@ -6,17 +6,14 @@
...
-    repo: "deb https://packages.elastic.co/logstash/{{ logstash_version }}/debian stable main"
+    repo: "deb https://artifacts.elastic.co/packages/5.x/apt stable main"
     state: present
+    update_cache: yes
 
 - name: Check if Logstash is already installed.
   stat: path=/etc/init.d/logstash
   register: logstash_installed
 
-- name: Update apt cache if repository just added.
```
Comment (senorsmile) [inline tasks/setup-Debian.yml:16]: why did we get rid of this?  It seems essential.

**Review by senorsmile — COMMENTED** [review]
one comment needs attention

File: tasks/setup-Debian.yml (hunk 1 above)
Comment (zahodi) [reply tasks/setup-Debian.yml:16]: This is done in "Add Logstash repository" via "update_cache: yes" option.

Comment (senorsmile) [reply tasks/setup-Debian.yml:16]: Oh yeah, way better place for it!!!

**Review by senorsmile — APPROVED** [review]
:+1: lgtnm

**Pull request merged**